code/controllers/subsystem/vis_overlays.dm

```diff
@@ -47,7 +47,7 @@ SUBSYSTEM_DEF(vis_overlays)
 		var/cache_id = "\ref[overlay]@{[world.time]}"
 		unique_vis_overlays += overlay
 		vis_overlay_cache[cache_id] = overlay
-		. = overlay
+	. = overlay
```

Comment:
could you add `return overlay` at the end and in the early return instead? so it's more explicit

Reply:
Done